taxi_service/settings.py

```diff
@@ -39,11 +39,13 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "debug_toolbar",
     "taxi",
 ]
 
 MIDDLEWARE = [
     "django.middleware.security.SecurityMiddleware",
+    "debug_toolbar.middleware.DebugToolbarMiddleware",
```

Comment:
The `DebugToolbarMiddleware` is included in `MIDDLEWARE`. Similar to the `debug_toolbar` app, ensure this middleware is only enabled in a development environment.